spec/models/plate_type_spec.rb

```diff
@@ -10,6 +10,7 @@
     create(:plate_type, name: 'ABgene_0765', maximum_volume: 800)
     create(:plate_type, name: 'ABgene_0800', maximum_volume: 180)
     create(:plate_type, name: 'FluidX075', maximum_volume: 500)
+    create(:plate_type, name: 'FluidX05', maximum_volume: 350)
```

Comment:
These maximum_volume values don't appear to gel with the plate names. I would have thought a 0.5ml tube would hold 500ul rather than 350. Plus the story mentions a 0.7 one that holds 500.
Maybe add a comment at the top as to why this is like this if you understand it?